timetracker/vms/templates/vms/dashboard.html

```diff
@@ -4,13 +4,14 @@
   <h1>Dashboard</h1>
   <h2>Quick Links</h2>
   {% if request.user.is_staff %}
-    <a href="{% url 'vms:create-staff-agency' %}">create a staffing agency!</a>
+    <a href="{% url 'vms:create-staff-agency' %}">create a staffing agency!</a><br>
```

Comment:
Only superusers are able to create staffing agencies, and that will be done from the admin interface as we discussed with Nick, so this link is not needed.